pages/api/performances/nominate.js

```diff
@@ -36,28 +36,37 @@ export default async (req, res) => {
   // TODO for now we assume frontend will filter correctly and only
   // eligible awards will show up for validation
   try {
+    // First clear all the current nominations
+    await prisma.awardPerformance.deleteMany({
+      where: {
+        performance_id: parseInt(performanceID),
+        user_id: parseInt(judgeID) || currentUserID,
+      },
+    });
+
     const awardPerformances = await prisma.$transaction(
       awardIDs.map(awardID =>
         prisma.awardPerformance.upsert({
           where: {
             unique_nomination: {
               award_id: awardID,
-              performance_id: performanceID,
-              user_id: userID,
+              performance_id: parseInt(performanceID),
+              user_id: parseInt(judgeID) || currentUserID, // Look for nominations matching the judge. If no judge user ID provided, use the current user
             },
           },
           create: {
             award_id: awardID,
-            performance_id: performanceID,
-            user_id: userID,
+            performance_id: parseInt(performanceID),
+            user_id: parseInt(judgeID) || currentUserID,
           },
           update: {},
         })
       )
     );
 
     return res.status(200).json(awardPerformances);
-  } catch {
+  } catch (err) {
+    console.log(err);
```

Comment:
Remove before merging

Reply:
This is in backend, so I'd assume it's okay. cc @thepenguinco